hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -175,7 +175,8 @@ class WorkflowParams(param.Parameterized):
     regression_metrics: str = param.String(default=None, doc="A list of names of metrics to compare")
     run_inference_only: bool = param.Boolean(False, doc="If True, run only inference and skip training after loading"
                                                         "model weights from the specified checkpoint in "
-                                                        "`src_checkpoint` flag. If False, run training and inference.")
+                                                        "`src_checkpoint` flag. Inference is run on both validation "
```

Comment:
Could we have some more explanation somewhere on why inference is done on validation set?

Reply:
We need it for the experiments we're running at the moment. For example, whole slide inference (that is quite expensive) we sometimes run it  only after training is finished in inference only mode and would like to check validation results as well